syne_tune/optimizer/schedulers/scheduler_searcher.py

```diff
@@ -42,7 +42,7 @@ def __init__(self, config_space: Dict[str, Any], **kwargs):
         # Generator for random seeds
         random_seed = kwargs.get("random_seed")
         if random_seed is None:
-            random_seed = np.random.randint(0, 2**32)
+            random_seed = np.random.randint(0, 2**31 - 1)
```

Comment:
Thanks!

I'd actually prefer to have a function like this in `syne_tune/optimizer/random_seeds.py`:

```
def generate_random_seed(random_state=np.random) -> int:
   return random_state.randint(0, 2**31 - 1)
```

Then, use this to implement both `RandomSeedGenerator.__call__` and sample the master random seed elsewhere.

Reply:
Then, we have the logic in one place.

Reply:
I've also submitted #645 with some further fixes along these lines.